model/postgres/postgres_server.rb

```diff
@@ -46,6 +47,15 @@ def configure_hash
       lc_time: "'C.UTF-8'"
     }
 
+    if primary?
+      configs[:archive_mode] = "on"
+      configs[:archive_timeout] = "60"
+      configs[:archive_command] = "'/usr/bin/wal-g wal-push %p --config /etc/postgresql/wal-g.env'"
+    else
+      configs[:recovery_target_time] = "'#{resource.restore_target}'"
+      configs[:restore_command] = "'/usr/bin/wal-g wal-fetch %f %p --config /etc/postgresql/wal-g.env'"
+    end
```

Comment:
Does `merge`, or `merge!` clean this up a bit?

Reply:
Depends on your definition of clean. Adds 4 new lines because `configs.merge({`  and its closing parentheses are on separate lines. However it removes repeated `configs` keyword.

Though, the funny thing is that when merge! is used, rubocop reverts back to this form for the else branch due to [this](https://msp-greg.github.io/rubocop-performance/RuboCop/Cop/Performance/RedundantMerge.html). I guess it is indeed more performant because merge needs to iterate the whole list. We can disable it or increase its threshold.

Reply:
Huh, okay, let's keep it as-is, though the first stanza could be `merge!` and abide the rule, I think a more similar notation makes sense for similar acts.